src/docs/releases/2.0.0.md

```diff
@@ -269,7 +269,52 @@ Here are the updated signatures:
    **To:**
    `Task<IDisplayResult> UpdateAsync(TSection section, UpdateEditorContext context)`
 
-These adjustments ensure compatibility and adherence to the latest conventions within the `SectionDisplayDriver` class.
+These adjustments ensure compatibility and adherence to the latest conventions within the `DisplayDriver` class.
+
+Additionally, we've enhanced display performance by streamlining the display drivers. To achieve this, the following methods have been marked as obsolete:
+
+- `IDisplayResult Display(TModel model)`
+- `IDisplayResult Edit(TModel model)`
+
+and removed the following methods:
+
+- `Task<IDisplayResult> DisplayAsync(TModel model, IUpdateModel updater)`
+- `IDisplayResult Display(TModel model, IUpdateModel updater)`
+- `Task<IDisplayResult> UpdateAsync(TModel model, IUpdateModel updater)`
+- `Task<IDisplayResult> EditAsync(TModel model, IUpdateModel updater)`
+- `IDisplayResult Edit(TModel model, IUpdateModel updater)`
+
+At the same time, we've introduced the following static methods in the `DisplayDriverBase`:
+
+- `Task<IDisplayResult> CombineAsync(params IDisplayResult[] results)`
+- `Task<IDisplayResult> CombineAsync(IEnumerable<IDisplayResult> results)`
+
+Similar refactoring was done to `SectionDisplayDriver` class. The following methods have been marked as obsolete:
+
+- `string PropertyName => typeof(TSection).Name;` instead you should use `SectionName`.
+- `Display(TSection section, BuildDisplayContext context)`
+- `Display(TSection section)`
+- `Edit(TSection section, BuildEditorContext context)`
+- `UpdateAsync(TSection section, UpdateEditorContext context)`
+- `UpdateAsync(TSection section, IUpdateModel updater, string groupId)`
+
+and removed the following methods:
+
+- `UpdateAsync(TModel model, TSection section, IUpdateModel updater, UpdateEditorContext context)`
+- `UpdateAsync(TSection section, IUpdateModel updater, UpdateEditorContext context)`
+- `UpdateAsync(TSection section, IUpdateModel updater, string groupId)`
+- `UpdateAsync(TSection section, UpdateEditorContext context)`
+- `EditAsync(TSection section, BuildEditorContext context)`
+- `DisplayAsync(TSection section, BuildDisplayContext context)`
+
+!!! note
+    To enhance your project's performance, address any warnings generated by the use of these obsolete methods. Please note that these methods will be removed in the next major release.
+
+A new `SiteDisplayDriver` base class was introduced to simplify the process of adding a UI settings.
+ 
+
+ !!! note
+    With the new `SiteDisplayDriver` base class for custom settings, you no longer need explicit checks like `context.GroupId.Equals(GroupId, StringComparison.OrdinalIgnoreCase)` as the base class handles this for you.
```

Comment:
`SettingsGroupId` needs to be overridden for this, which is missing from the below samples.

Reply:
both implementation in the 2.0.0 release of `SiteDisplayDriver` are stripped versions. Both state the following

```
    //
    // For  example simplicity, other methods are not visible.
    //
```